lib/commands/config/index.js

```diff
@@ -46,7 +46,7 @@ class ConfigCommand extends Command {
                     }));
                 }
 
-                this.instance.config.set(configKey, value);
+                this.instance.config.set(configKey, option.transform ? option.transform(value) : value);
```

Comment:
I wonder if transform shouldn't be called before validate? I'm open to either place but I feel like transforming before validation would be a bit better? That way transform doesn't leave us with an invalid value for whatever reason.